.gitignore

```diff
@@ -72,3 +72,6 @@ typings/
 
 # Test cafe reports
 report*.json
+
+# IDE configs
+.idea
```

Comment:
Maybe this could live in your global `.gitconfig`? Feels more specific to the developers chosen environment that related to this application.